packages/enzyme-mocha/src/index.js

```diff
@@ -0,0 +1,33 @@
+let jsdomify;
+
+try {
+  jsdomify = require('jsdomify');
+  jsdomify.create();
+  require('react'); // require react explicitly after jsdomify.create() has been called
+  jsdomify.destroy();
+} catch (e) {
+  // jsdom is not supported
+}
+
+export function jsdomSetup() {
+  if (!jsdomify) return;
+  jsdomify.create();
+}
+
+export function jsdomTeardown() {
+  if (!jsdomify) return;
+  jsdomify.destroy();
+}
+
+export function describeWithDOM(a, b) {
+  describe('(uses jsdom)', () => {
```

Comment:
the function retains a lexical reference to the one that's in the global namespace when the function is defined, it's not freshly looked up. do you mean this to be `global.describe` then?